packages/eslint-checker/src/esconfig-helpers.ts

```diff
@@ -0,0 +1,51 @@
+import { ESLint, Linter } from 'eslint';
+import { CosmiconfigResult } from 'cosmiconfig/dist/types';
+
+// Override some compiler options that have to do with code quality. When mutating, we're not interested in the resulting code quality
+// See https://github.com/stryker-mutator/stryker-js/issues/391 for more info
+const LINT_RULES_OVERRIDES: Readonly<Partial<Record<string, string>>> = Object.freeze({
+  'no-unused-vars': 'off',
+  'import/no-unresolved': 'off',
+  '@typescript-eslint/no-unused-vars': 'off',
+  '@typescript-eslint/no-empty-function': 'off',
+});
+
+/**
+ * Overrides some options to speed up compilation and disable some code quality checks we don't want during mutation testing
+ * @param parsedConfig The parsed config file
+ */
+function overrideOptions(parsedConfig: { config?: any }): Omit<Linter.Config, 'extends' | 'plugins'> {
+  const rules = {
+    ...parsedConfig.config?.rules,
+    ...LINT_RULES_OVERRIDES,
+  };
+
+  const config = {
+    ...parsedConfig.config,
+    rules,
+  };
+
+  if (config.extends) {
```

Comment:
I think this if check can be removed:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#description

>If the property which you are trying to delete does not exist, delete will not have any effect and will return true.